keywording/text.xml

```diff
@@ -267,6 +291,38 @@ Vulnerability Treatment Policy</uri>
 
 </body>
 
+<subsection>
+<title>Stabilization rules</title>
+<body>
+
+<p>
+SPARC: You must have prior permission from the arch lead. Usually we expect
+you to be on the sparc alias for QA reasons, although other arrangements
+can be made if you will only be working with a small group of packages.
+</p>
+
+<p>
+ALPHA: Maintainers may keyword their own packages but are reminded to inform
+the Alpha team if they can help out with testing and keywording packages so
+the team can keep an eye out for possible keywording mistakes.
+</p>
+
+<p>
+Exotic architectures (like alpha, ia64, sparc, hppa, ppc*) are short
+on manpower, so it's best if you avoid opening stabilization bugs for
```

Comment:
That's going to be confusing. It should talk of adding stable keywords to new packages; we still want people to stabilize new versions of existing packages.

Reply:
I've changed it to say "avoid opening bugs for stabilization of new packages".